src/service/capabilityService.ts

```diff
@@ -15,5 +16,29 @@ export const capabilityService = {
         catch (e) {
             throw new Error("Failed to get job capabilities")
         }
+    },
+
+    async createCapability(createCapabilityRequest : CreateCapabilityRequest, token? : string): Promise<void> {
+
+        if (!token) throw new Error("You are not logged in (no token provided)")
+
+        try {
+            const response = await axios.post(`${process.env.API_URL}api/capabilities/`, createCapabilityRequest, { 
+                headers: { Authorization: `Bearer ${token}` }
+            });
+
+            if (response.status === 200) {
+                return;
+            } 
+            if(response.status === 400) {
+                throw new Error(`Bad request. ${response.data.errorMessage}`);
+            }
+            else {
+                throw new Error(response.data.errorMessage);
+            }
+        } catch (e) {
+            console.error(e);
```

Comment:
user wouldnt b able to see Error that thrown on 37. And error message would be lost and returned to "Failed creating capability" maybe add the error message to the caught and thrown new error on line 42